bpf/process/bpf_execve_event.c

```diff
@@ -355,6 +375,8 @@ execve_send(void *ctx)
 			curr->bin.path_length--;
 		}
 #endif
+
+		update_mb_bitset(curr);
```

Comment:
any particular reason to do it here at the very end or just, why not?

Reply:
update_mb_bitset needs ->binary to be set for the map lookup. So it happens just after we set it.